### DISCUSSION TITLE
how to add embedded youtube movie

### DISCUSSION THREAD

**Discussion by jfvdvelden** [gethinode/hinode#606] [Help]
prob 3:

My youtube movies are aslo not showed anymore.

I cant find any docs about this.

any tips?

<center><a href="https://www.youtube.com/watch?feature=player_embedded&v=QqLBbRQ-B5w
" target="_blank"><img src="https://img.youtube.com/vi/QqLBbRQ-B5w/0.jpg" 
alt="AGRITOURISM DOL IN GOLOGORIČKI DOL" width="240" height="180" border="10" /></a></center>

**Comment by markdumay** [comment]
Please check the answers to #604 and #605.
